doc/howToUse.rst

```diff
@@ -5,15 +5,15 @@ How to use eTraGo?
 
 After you installed eTraGo you would typically start optimization runs by
 executing the ‘appl.py’ which is situated in 
-``./eTrago/etrago/`` (e.g by ``python3 appl.py``).
+``./eTrago/etrago/`` (e.g by ``python3 appl.py`` from the terminal).
 
 eTraGo doesn't have a graphical user interface, 
 the ‘appl.py’ is used as a simple user interface which can be edited with 
 the preferred python-editor.
 Here parameters, calculation methods and scenario settings are set in a python
 dictionary called 'args'. 
-To run the desired calculation, it is crucial to understand these parameters. 
-In addition, some of them contradict the usage of others. 
+To run the desired calculation, it is crucial to understand these parameters, 
+since for particular cases some of them contradict the usage of others. 
```

Comment:
I'm not sure if I understand why this was changed. It is still also necessary to understand the parameters, not only because they can contradict each others, right?

Reply:
Agree. I proceed to revert this change.